packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -68,6 +69,13 @@ export default function globalReferenceToImport(
     return fileForIdentifier[identifier];
   }
 
+  function searchForNamedExports(filename) {
+    if (filename.endsWith('.coffee'))
+      return false;
+    const namedExports = j(fs.readFileSync(filename).toString())
```

Comment:
Do we have ay strings where that's the case? You can always force it to appear at the start of a line, which again, won't catch every case, but might be good enough if it gives us significant speed improvements.